openrpc.yaml

```
@@ -285,7 +286,8 @@ methods:
     description: >-
       Presents a data message for the user to sign in a structured and readable
       format and returns the signed response. Introduced by
-      [EIP-712](https://eips.ethereum.org/EIPS/eip-712).
+      [EIP-712](https://eips.ethereum.org/EIPS/eip-712). 
+      This method requires that the user has granted permission to interact with their account first, so make sure to call `eth_requestAccounts` or `wallet_requestPermissions` first.
```

```suggestion
      This method requires that the user has granted permission to interact with their account first, so make sure to call `eth_requestAccounts` first.
```

We recommend using `eth_requestAccounts` for all wallet restricted methods, so we don't need to specify `wallet_requestPermissions` in the description here. If the user clicks onto `eth_requestAccounts`, they should see that it calls `wallet_requestPermissions` internally.